nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -146,11 +146,11 @@ define([
                 tabs: {
                     configure: {
                         label: 'Configure',
-                        widget: ConfigureWidget,
+                        widget: ConfigureWidget.editMode,
```

Comment:
The ConfigureWidget now implements 2 "widgets" - the edit and view versions. This lets the bulkImportCell.runTab not have to care about what widget its running and what mode it should be in.

Reply:
That's great! Be fantastic to port that over to the app cell and get rid of those vast chunks of copy-pasta